lib/amorail/client.rb

```diff
@@ -21,12 +29,19 @@ def connect
       @connect || self.class.new
     end
 
-    def authorize
+    def authorize(args = {})
       self.cookies = nil
+
+      user_login, user_hash = if Amorail.client.usermail.present? && Amorail.client.api_key.present?
+                                [Amorail.client.usermail, Amorail.client.api_key]
+                              else
+                                [args[:usermail].to_s, args[:api_key].to_s]
+                              end
+
       response = post(
-        Amorail.config.auth_url,
-        'USER_LOGIN' => Amorail.config.usermail,
-        'USER_HASH' => Amorail.config.api_key
+          Amorail.config.auth_url,
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.